provd/internal/services/manager_test.go

```diff
@@ -0,0 +1,71 @@
+package services_test
+
+import (
+	"context"
+	"flag"
+	"testing"
+
+	"github.com/canonical/ubuntu-desktop-provision/provd/internal/services"
+	"github.com/canonical/ubuntu-desktop-provision/provd/internal/testutils"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+)
+
+func TestNewManager(t *testing.T) {
+	tests := map[string]struct {
+		wantErr bool
+	}{
+		"Successfully creates the manager.": {},
```

Comment:
ah, you assumed a dot here, we didn’t use them otherwise all subtests name will end with a dot, but that’s as you think may fit.

Reply:
Ah I see, I'll revert to the way the other subtests are named :slightly_smiling_face: